### PR TITLE
test: add basic tests for loan offer #create

### PR DESCRIPTION
**Pull request by graemecode**

### CODE REVIEW

File: __test__/integration/loan_offer/runners/create.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Dharma } from "../../../../src";
+
+import { LoanOffer } from "../../../../src/types";
+
+export async function testCreate(dharma: Dharma, params: any) {
+    describe("passing valid params", () => {
+        let loanOffer: LoanOffer;
+
+        beforeAll(async () => {
+            loanOffer = await LoanOffer.create(dharma, params) as LoanOffer;
```
Comment (saturnial) [inline __test__/integration/loan_offer/runners/create.ts]: so this should be: `LoanOffer.create<LoanOffer>(dharam, params)`

Comment (graemecode) [reply __test__/integration/loan_offer/runners/create.ts]: oh of course! who could forget?

File: __test__/integration/loan_offer/runners/create.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export async function testCreate(dharma: Dharma, params: any) {
+    describe("passing valid params", () => {
+        let loanOffer: LoanOffer;
+
+        beforeAll(async () => {
+            loanOffer = await LoanOffer.create<LoanOffer>(dharma, params);
+        });
+
+        test("not signed by the debtor", async () => {
+            const isSignedByDebtor = await loanOffer.isSignedByDebtor();
```
Comment (saturnial) [inline __test__/integration/loan_offer/runners/create.ts]: this test should just test that loanOffer is an instanceOf LoanOffer

Comment (graemecode) [reply __test__/integration/loan_offer/runners/create.ts]: that's just a redundant test though. typescript already knows this

**Pull request merged**